web-server/app/views/searchResults.scala.html

```diff
@@ -1,29 +1,25 @@
-@import codesearch.core.search.Search.PackageResult
-
+@import codesearch.core.search.{ErrorResponse, Response, SuccessResponse}
 @(
-        updated: String,
-        packages: Seq[PackageResult],
-        query: String,
-        filter: Option[String],
-        filePath: Option[String],
-        insensitive: Boolean,
-        space: Boolean,
-        precise: Boolean,
-        sources: Boolean,
-        page: Int,
-        totalMatches: Int,
-        callURI: String,
-        lang: String
+response: Response
 )
 
-    @headExtra = {
-        <script src="@routes.Assets.versioned("prismjs.js")"></script>
-        <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("css/prismjs.css")">
-        <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("css/search.css")">
-    }
+@headExtra = {
+<script src="@routes.Assets.versioned(" prismjs.js")"></script>
+    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("css/prismjs.css")">
+    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("css/search.css")">
+}
 
-    @wrapper(s"Codesearch | $lang", headExtra) {
-        @searchBox(s"/$lang/search", query, filter, filePath, insensitive, space, precise, sources)
-        @resultFrame(lang, insensitive, space, precise, query, updated, packages, totalMatches)
-        @pagination(page, totalMatches, callURI)
+@response match {
+    case SuccessResponse(updated, packages, query, filter, filePath, insensitive, space, precise, sources, page, totalMatches, callURI, lang) => {
+        @wrapper(s"Codesearch | $lang", headExtra) {
+            @searchBox(s"/$lang/search", query, filter, filePath, insensitive, space, precise, sources)
+            @resultFrame(lang, insensitive, space, precise, query, updated, packages, totalMatches)
+            @pagination(page, totalMatches, callURI)
+        }
+    }
+    case ErrorResponse(message) => {
+      @wrapper(s"Codesearch ", headExtra) {
```

Comment:
Replace useless string-interpolated s"Codesearch " to plain string - "Codesearch".

Reply:
By the way, you can use [suggestions](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) for those small changes – Cmd+G or Ctrl+G.

```suggestion
      @wrapper("Codesearch", headExtra) {
```